foe_foundry/creatures/_yaml_template.py

```
@@ -46,27 +87,738 @@ def __init__(self, yaml_data: dict):
+            if template_key == "wolf":
+                if monster_key == "wolf":
+                    return PowerSelection(powers_module.LoadoutWolf)
+                elif monster_key == "dire-wolf":
+                    return PowerSelection(powers_module.LoadoutDireWolf)
+                elif monster_key == "winter-wolf":
+                    return PowerSelection(powers_module.LoadoutFrostWolf)
+                elif monster_key == "fellwinter-packlord":
+                    return PowerSelection(powers_module.LoadoutPacklord)
+                else:
+                    return PowerSelection(powers_module.LoadoutWolf)
+            elif template_key == "animated-armor":
+                if monster_key == "animated-armor":
+                    return PowerSelection(powers_module.LoadoutAnimatedArmor)
+                elif monster_key == "animated-runeplate":
+                    return PowerSelection(powers_module.LoadoutRunicSpellplate)
+                else:
+                    return PowerSelection(powers_module.LoadoutAnimatedArmor)
+            else:
+                # For other templates, try to find a default loadout
+                # Look for common loadout names
+                loadout_attrs = [attr for attr in dir(powers_module) 
+                               if attr.startswith('Loadout') and not attr.endswith('__')]
+                if loadout_attrs:
+                    # Use the first loadout found
+                    loadout = getattr(powers_module, loadout_attrs[0])
+                    return PowerSelection(loadout)
+                
+                # If no loadout found, return empty selection
+                return PowerSelection([])
+                
+        except (ImportError, AttributeError) as e:
+            # If powers module doesn't exist or loadout not found, return empty selection
+            print(f"Warning: Could not load powers for template {template_key}: {e}")
```

this should instead raise an error, and we can add the appropriate aliases for the places where it doesn't work